meshnets/main.py

```diff
@@ -1,45 +1,65 @@
 """The main file to run our code
 
-Currently, it loads mesh data from a .vtk file, defines a wind vector,
-and creates a Graph object from them
+Currently, it processes mesh data to graph data and creates training
+and testing datasets and dataloaders from the processed '.pt' files.
 """
 
 import os
+from pathlib import Path
 
 from absl import app, flags, logging
-import numpy as np
+import torch
+from torch.utils.data import random_split
+from torch_geometric.loader import DataLoader
 
-from meshnets.utils import data_loading
 from meshnets.utils import data_processing
+from meshnets.utils.datasets import FromDiskDataset
 
 FLAGS = flags.FLAGS
-flags.DEFINE_string('input_object', os.path.join('data', 'blob.vtk'),
-                    'File path of the object to be visualized')
+flags.DEFINE_string('raw_data_dir', os.path.join('data', 'vtk'),
```

Comment:
What does raw mean?

Reply:
If it is just do differentiate between vtk and pt data, it will be best to have `vtk_data_dir` and `pt_data_dir`.

Reply:
"raw" are the mesh files. Here they are .vtk but could also be .obj. I can rename raw to mesh if you think it's more clear

Reply:
mesh is good!